plugins/ingest-attachment/src/test/java/org/elasticsearch/ingest/attachment/AttachmentProcessorTests.java

```diff
@@ -229,6 +229,7 @@ public void testParseAsBytesArray() throws Exception {
 
         Map<String, Object> document = new HashMap<>();
         document.put("source_field", bytes);
+        document.put("resource_name", path);
```

Comment:
Let's remove this line so these tests remain unchanged as well.

Reply:
but, the new field "resource_name"  should be added here, if not , the "resource_name" will meaningless

Reply:
Please see my other comment here: https://github.com/elastic/elasticsearch/pull/64389#issuecomment-739958085 -- I would like to isolate the testing for `resource_name` to a new test method and keep the existing tests to verify that the processor's behavior doesn't change if `resource_name` is unspecified. Let me know if you have any questions about that.